cluster/ci/origin/config/command-tests.yaml

```diff
@@ -0,0 +1,141 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: command-tests
+parameters:
+- description: Name of the build to associate with the source image and artifacts
+  displayName: Build Name
+  name: BUILD_NAME
+  required: true
+- description: The reference to the image that contains the compiled binaries to test
+  displayName: Binaries Image Reference
+  name: BINARY_IMAGE_REF
+  required: true
+- description: The image pull secret of the default service account
+  displayName: Image Pull Secret
+  name: PULL_SECRET_NAME
+  required: true
+- description: The id of the run
+  displayName: Run ID
+  name: RUN_ID
+  required: true
+- name: REPO_OWNER
+  required: true
+  value: "openshift"
+- name: REPO_NAME
+  required: true
+  value: "origin"
+- name: JOB_NAME
+  required: true
+- name: BUILD_NUMBER
+  required: true
+- name: PULL_BASE_REF
+  required: true
+  value: ""
+- name: PULL_BASE_SHA
+  required: true
+  value: ""
+- name: PULL_REFS
+  required: true
+  value: ""
+- name: PULL_NUMBER
+  value: ""
+- name: PULL_PULL_SHA
+  value: ""
+objects:
+- apiVersion: v1
+  kind: Pod
+  metadata:
+    generateName: "cmd-${BUILD_NUMBER}-${BUILD_NAME}-"
+    labels:
+      build: "${BUILD_NAME}"
+      test: command-tests
+      run: "${RUN_ID}"
+      created-by-prow: "true"
+  spec:
+    initContainers:
+    - name: save-config
+      env:
+      - name: JOB_NAME
+        value: ${JOB_NAME}
+      - name: BUILD_NUMBER
+        value: ${BUILD_NUMBER}
+      - name: REPO_OWNER
+        value: ${REPO_OWNER}
+      - name: REPO_NAME
+        value: ${REPO_NAME}
+      - name: PULL_BASE_REF
+        value: ${PULL_BASE_REF}
+      - name: PULL_BASE_SHA
+        value: ${PULL_BASE_SHA}
+      - name: PULL_REFS
+        value: ${PULL_REFS}
+      - name: PULL_NUMBER
+        value: ${PULL_NUMBER}
+      - name: PULL_PULL_SHA
+        value: ${PULL_PULL_SHA}
+      command:
+      - release-ci
+      - save-config
+      - --config-path=/var/run/logging-config/logging-config.json
+      image: "${BINARY_IMAGE_REF}"
+      imagePullPolicy: IfNotPresent
+      volumeMounts:
+      - mountPath: /var/run/logging-config
+        name: logging-config
+      - mountPath: /var/run/logging
+        name: logging
+    containers:
+    - command:
+      - release-ci
+      - entrypoint
+      - --config-path=/var/run/logging-config/logging-config.json
+      - --
+      - /bin/bash
+      - -c
+      - 'ARTIFACT_DIR=/var/run/logging/output LOG_DIR=/var/run/logging/output/logs JUNIT_REPORT=true SKIP_TEARDOWN=1 HOME=/tmp HOSTNAME=$(hostname) TERM=xterm-256color make test-cmd -k'
```

Comment:
instead of setting `$ARTIFACT_DIR` and `$LOG_DIR` just set `$OS_OUTPUT_BASEPATH=/var/run/logging`